compiler/src/middle_end/linearize.re

```diff
@@ -1557,6 +1521,33 @@ let rec transl_anf_statement =
         ],
       );
     };
+  | TTopExport(exports) =>
+    List.iter(
+      ({tex_path}) => {
+        let {val_fullpath, val_type, val_repr} =
+          Env.find_value(tex_path, env);
+        switch (val_fullpath) {
+        | Path.PExternal(Path.PIdent(mod_) as p, ident, _) =>
+          let allocation_type = get_allocation_type(env, val_type);
+          let mod_decl = Env.find_module(p, None, env);
+          // Lookup external exports to import them into the module
```

Comment:
I'm a little confused what this is doing. Can you clarify a little more?

Reply:
Yeah! So the way imports actually make it into the WebAssembly module is via `lookup_symbol` registering the individual values and functions to include them in the module. Previously doing `export foo` was sugar for `export let foo = foo`, making `foo` get used and thus be included as an import, but now an `export` statement skips the extra binding. The lookup here is effectively us saying that we plan to use that value so it actually gets imported.